app/views/patients/_new_patient.html.erb

```diff
@@ -1,54 +1,66 @@
+    </div>
+    <div class="col-4">
+      <%= f.text_field :name,
+                       label: t('common.name'),
+                       value: name,
+                       required: true %>
+    </div>
 
-      <div class="col-sm-2">
-        <%= f.select :last_menstrual_period_weeks, options_for_select(weeks_options, patient.last_menstrual_period_weeks ), label: t('patient.dashboard.weeks_along'), autocomplete: 'off' %>
-      </div>
-      <div class="col-sm-2">
-        <%= f.select :last_menstrual_period_days, options_for_select(days_options, patient.last_menstrual_period_days ), label: t('common.days_along'), autocomplete: 'off' %>
-      </div>
+    <div class="col">
+      <%= f.select :last_menstrual_period_weeks,
+                   options_for_select(weeks_options, patient.last_menstrual_period_weeks),
+                   label: t('patient.dashboard.weeks_along'),
+                   autocomplete: 'off' %>
+    </div>
+    <div class="col">
+      <%= f.select :last_menstrual_period_days,
+                   options_for_select(days_options, patient.last_menstrual_period_days),
+                   label: t('common.days_along'),
+                   autocomplete: 'off' %>
+    </div>
 
-      <div class="col-sm-2">
-        <%= f.select :language, options_for_select(language_options, patient.language ),
-          label: t('patient.information.language'), autocomplete: 'off' %>
-      </div>
+    <div class="col">
+      <%= f.select :language,
+                   options_for_select(language_options, patient.language),
+                   label: t('patient.information.language'),
+                   autocomplete: 'off' %>
     </div>
+  </div>
 
-    <div class="row">
-      <div class="col-md-2">
-        <%= f.date_field :initial_call_date, label: t('patient.new.initial_call_date'), value: today,  required: "required" %>
-      </div>
+  <div class="row">
+    <div class="col">
+      <%= f.date_field :initial_call_date,
+                       label: t('patient.new.initial_call_date'),
+                       value: today,
+                       required: "required" %>
+    </div>
 
-      <div class="col-sm-4">
-        <%= f.select :voicemail_preference,
-                     options_for_select(voicemail_options),
-                     label: t('patient.information.voicemail_preference') %>
-      </div>
+    <div class="col">
+      <%= f.select :voicemail_preference,
+                   options_for_select(voicemail_options),
+                   label: t('patient.information.voicemail_preference') %>
+    </div>
 
-      <div class="col-sm-2">
-        <%= f.select :line, options_for_select(lines, current_line) %>
-      </div>
+    <div class="col">
+      <%= f.select :line, options_for_select(lines, current_line) %>
+    </div>
 
-      <div class="col-sm-2 btn-space">
-        <%= button_tag class: 'btn btn-primary', id: 'create_new_patient_button' do %>
```

Comment:
I have no idea how this ever worked, so this actually is a somewhat meaningful change.